app/controllers/api/posts_controller.rb

```diff
@@ -3,4 +3,19 @@ def index
     posts = Post.all
     render json: { posts: posts }
   end
+
+  def create
+    post = Post.create(post_params)
+    if post.persisted?
+      render json: { message: 'Your post was successfully created!' }, status: 201
```

Comment:
maybe change this message to something fun. we are probably not going to use it in the front end so go nuts!

Reply:
@ViktorHek  We'll take this into consideration!